projects/plugins/jetpack-mu-wpcom/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
+### This is a list detailing changes for all jetpack-mu-wpcom releases.
```

Comment:
Why this instead of the other boilerplate?

Reply:
Ah, https://github.com/Automattic/jetpack/pull/27815#issuecomment-1349680890

Reply:
Updated in c20c485